src/map/clif.c

```diff
@@ -13258,6 +13248,17 @@ static void clif_parse_UseSkillToPosSub(int fd, struct map_session_data *sd, uin
 		return;
 	}
 
+	pc->autocast_set_current(sd, skill_id);
```

Comment:
here look like same movement as in previous comment

Reply:
Undid it, same as above